contentprovider.go

```diff
@@ -255,7 +254,7 @@ func findSection(secs []DocumentSection, off, sz uint32) *DocumentSection {
 	return nil
 }
 
-func matchScore(secs []DocumentSection, m *LineMatch) float64 {
+func matchScore(m *LineMatch) float64 {
```

Comment:
don't change this function, lets just keep it the same as upstream to simplify merging. Rather just always pass nil in for secs.